src/audio/sink_api_helper.c

```diff
@@ -185,3 +185,9 @@ size_t sink_get_min_free_space(struct sof_sink *sink)
 {
 	return sink->min_free_space;
 }
+
+uint32_t sink_get_id(struct sof_sink *sink)
```

Comment:
These should be static inline. There is PR in flow for splitting header directories and we will be able to catch this in the correct folder i.e. for baseFW or module usage.

Reply:
static inline and put it tou sink_api.h and source_api.h accordingly.

Reply:
yes. We agreed lately the full sink/src encapsulation is too expensive and all "small" functions should be inline (https://github.com/thesofproject/sof/issues/8100)

It has been implemented for sink/src as a part of https://github.com/thesofproject/sof/pull/8365. Till 8365 is merged I prefer to follow the obsolete way to avoid merge conflicts